src/v2/view-builder/views/google-authenticator/EnrollGoogleAuthenticatorBarcodeView.js

```diff
@@ -0,0 +1,34 @@
+import { View } from 'okta';
+import hbs from 'handlebars-inline-precompile';
+
+export default View.extend({
+  className: 'oie-enroll-google-authenticator-barcode',
+  template: hbs`
+      {{#if href}}
+      <div class="google-authenticator-setup-info-title">{{i18n code="oie.enroll.google_authenticator.scanBarcode.title" bundle="login"}}</div>
+      <div class="qrcode-info-container">
+        <p class="google-authenticator-setup-info">
+          {{i18n code="oie.enroll.google_authenticator.scanBarcode.description" bundle="login"}}
+        </p>
+        <div class="qrcode-container">
+          <img class="qrcode" src={{href}} alt="qr code"></img>
+          <a href="#" class="cannot-scan-link">
+            {{i18n code="oie.enroll.google_authenticator.scanBarcode.cannotScan" bundle="login"}}
+          </a>
+        </div>
+        </div>
+      {{/if}}
+    `,
+
+  getTemplateData () {
+    const contextualData = this.options.appState.get('currentAuthenticator').contextualData;
+    return { href: contextualData.qrcode?.href };
+  },
+
+  postRender () {
+    this.$el.find('.cannot-scan-link').on('click', (event) => {
+      event.preventDefault();
+      this.options.model.trigger('triggerBarcodeView');
```

Comment:
check is there is state here or try to trigger on appState if possible.

Reply:
If this a the form link then these part of the code should auto handle this scenario.. 
https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/components/Link.js#L39

Reply:
@pradeepdewda-okta  this isnt a form link. We just hide. a part of the view and show another and do not rerender form here.

Reply:
- Might prefer trigger on Model given it's `local` state (only local to google authn enroll view)
- Are you able to use the model events as well?
```
modelEvents: {
  'click .cannot-scan-link', 'triggerBarcodeView'
},

triggerBarcodeView() {
  this.options.model.trigger('triggerBarcodeView');
}
```